department-of-veterans-affairs/abd-vro
# MCP-2167 change order of claims to be sorted by updatedAt instead of claimId

Change order of claims to be sorted by updatedAt instead of claimId



## What was the problem?


Associated tickets or Slack threads:

- [MCP-2167](https://amida.atlassian.net/browse/MCP-2167)

## How does this fix it?[^1]

Made the service return the claims in a sorted list where it is sorted by the updatedAt field, meaning we are sorting by the latest updated claims.  Fixed tests as well so that createdAt and updatedAt fields are not null during testing.

## How to test this PR
- Step 1 Run the full health assessment with different claimSubmissionIds and veteran ICNs.
- Step 2[^secrel] Verify the latest inserted claims are at the top of the list returned.


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
